botx/models/messages/sending/message.py

```diff
@@ -24,13 +24,14 @@
     "{0} can not be passed along with manual validated_values for it"
 )
 
-EMBED_MENTION_TEMPATE = (
+EMBED_MENTION_TEMPLATE = (
     "<embed_mention:{mention_type}:"
     "{mentioned_entity_id}:{mention_id}:{mention_name}>"  # noqa: WPS326
 )
 EMBED_MENTION_RE: Final = re.compile(
-    "<embed_mention:(?P<mention_type>.+?):(?P<mentioned_entity_id>.+?)"
-    ":(?P<mention_id>.+?):(?P<name>.+?)?>",  # noqa: WPS326 C812
+    "<embed_mention:(?P<mention_type>.+?):"
+    r"(?P<mentioned_entity_id>[0-9a-f\-]+?):"
+    r"(?P<mention_id>[0-9a-f\-]+?):(?P<name>.+?)??>"  # noqa: WPS326 C812
```

Comment:
May be need to add some unit-tests checking this regexp works correctly for each case. 
E.g. with optional <name> group and without it.